modules/recording/recording/libs/download.ts

```diff
@@ -0,0 +1,98 @@
+import { PDFDocument, rgb, StandardFonts } from "pdf-lib";
+import { saveAs } from "file-saver";
+
+class Download {
+
+  static async downloadAudio(audioUrl: string) {
+    try {
+      const response = await fetch(audioUrl);
+      const blob = await response.blob();
+      const blobUrl = URL.createObjectURL(blob);
+
+      const anchor = document.createElement("a");
+      anchor.href = blobUrl;
+      anchor.download = `audio-${new Date().toLocaleDateString()}.mp3`;
+
+      document.body.appendChild(anchor);
+      anchor.click();
+      document.body.removeChild(anchor);
+
+      URL.revokeObjectURL(blobUrl);
+    } catch (error) {
+      console.error("Failed to download audio file:", error);
+      throw error;
```

Comment:
Catch and throw is `no-sense` 
Toast. the orginal error, hidding or silentling the original error can affect the user experience